node/setup.go

```diff
@@ -189,8 +189,16 @@ func logNodeStartupInfo(state sm.State, pubKey crypto.PubKey, logger, consensusL
 	}
 }
 
-func onlyValidatorIsUs(state sm.State, pubKey crypto.PubKey) bool {
+func hasMajorityVotingPower(state sm.State, pubKey crypto.PubKey) bool {
 	if state.Validators.Size() > 1 {
+		for _, val := range state.Validators.Validators {
+			if val.PubKey.Equals(pubKey) {
```

Comment:
Do we need to check for a nil pubkey (just cause we do further down the page)

Reply:
not sure, validators shouldn’t be able to have a nil pubkey, and I would expect that this is safe because the equals method should handle nil correctly. The only real reason to do a nil check as far as I can see is to avoid a panic.